scripts/gui/microbe_hud.mjs

```diff
@@ -481,27 +527,210 @@ function hideWinText(){
     document.getElementById("winContainer").style.display = "none";
 }
 
+// Variables needed for changes
+let barToChange = "";
```

Comment:
Why are these global variables?

Reply:
I put them as global because they are used in two different methods for compress and expands, i thought that put them as global wouldn't nice to see but avoid to duplicate them.

Reply:
Global variables are bad, and should be avoided whenever possible.
These can be easily be defined in the functions using them.
http://wiki.c2.com/?GlobalVariablesAreBad